### PR TITLE
Feature#24/my events view

### PR DESCRIPTION
**Pull request by javs1**
Added My events view and functionality
What:
 - Added view of the events created by the user
 - Given CRUD functionality 

 
 Why:
 - Let the user see what events are at their charge
 - Let the user Create, Read, Update and Delete those events
 
image

### CODE REVIEW

File: EasyLearn/app/views/events/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+      <td class="text-center" ><%= event.description%></td>
+      <td class="text-center" ><%= event.category.name%></td>
+      <td class="text-center"><%= event.day%></td>
+      <td class="text-center"><%= event.hour%></td>
+      <td class="text-center"><%= event.link%></td>
+      <td class="text-center">
+        <button type="button" id="" data-toggle="modal" data-target="#editEvent<%= event.id %>" class="btn btn-success">
+          <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-pencil-fill" viewBox="0 0 16 16">
+            <path d="M12.854.146a.5.5 0 0 0-.707 0L10.5 1.793 14.207 5.5l1.647-1.646a.5.5 0 0 0 0-.708l-3-3zm.646 6.061L9.793 2.5 3.293 9H3.5a.5.5 0 0 1 .5.5v.5h.5a.5.5 0 0 1 .5.5v.5h.5a.5.5 0 0 1 .5.5v.5h.5a.5.5 0 0 1 .5.5v.207l6.5-6.5zm-7.468 7.468A.5.5 0 0 1 6 13.5V13h-.5a.5.5 0 0 1-.5-.5V12h-.5a.5.5 0 0 1-.5-.5V11h-.5a.5.5 0 0 1-.5-.5V10h-.5a.499.499 0 0 1-.175-.032l-.179.178a.5.5 0 0 0-.11.168l-2 5a.5.5 0 0 0 .65.65l5-2a.5.5 0 0 0 .168-.11l.178-.178z"></path>
+          </svg>
```
Comment (YilKanda) [inline EasyLearn/app/views/events/index.html.erb]: What do you think about passing the SVG code to an external file?

File: EasyLearn/app/views/events/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+        </button>
+        <form method="post" action="/delete_event" class="btn">
+          <input type="hidden" name ="id" value="<%= event.id %>" id ="id" class="form-control" placeholder="privateNumber" method="post" action="/delete_event">
+          <button type="submit" class="btn btn-danger">
+            <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-x-circle-fill" viewBox="0 0 16 16">
+              <path d="M16 8A8 8 0 1 1 0 8a8 8 0 0 1 16 0zM5.354 4.646a.5.5 0 1 0-.708.708L7.293 8l-2.647 2.646a.5.5 0 0 0 .708.708L8 8.707l2.646 2.647a.5.5 0 0 0 .708-.708L8.707 8l2.647-2.646a.5.5 0 0 0-.708-.708L8 7.293 5.354 4.646z"></path>
+            </svg>
```
Comment (YilKanda) [inline EasyLearn/app/views/events/index.html.erb]: What do you think about passing the SVG code to an external file?

File: EasyLearn/app/views/events/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+            <div class="input-group form-group mb-3">
+              <%= f.collection_select(:category_id, Category.all, :id, :name) %>
+            </div>
+            <div class="form-group">
+              <%= f.submit  onClick: "window.location.reload();", class: "form-control" %>
+						</div>
+          <% end %>
+          </div>
+        </div>
+      </div>
```
Comment (YilKanda) [inline EasyLearn/app/views/events/index.html.erb]: What do you think about passing the modal code to an external file?

File: EasyLearn/app/views/events/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+            </div>
+            <div class="form-group">
+              <%= f.submit class:"btn btn-success float-right" %>
+						</div>
+          <% end %>
+          </div>
+        </div>
+      </div>
+    </div>
+<div>
```
Comment (YilKanda) [inline EasyLearn/app/views/events/index.html.erb]: What do you think about passing the modal code to an external file?

File: EasyLearn/config/database.yml.example [hunk 5]
```diff
@@ -4,8 +4,8 @@
 postgresql: &postgresql
   adapter: postgresql
   host: localhost
-  username: username
-  password: password
+  username: p_final
+  password: p_finalBC2021
```
Comment (YilKanda) [inline EasyLearn/config/database.yml.example]: This change is not necessary.

File: EasyLearn/app/views/events/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,132 @@
+<%= render "layouts/menu"%>
+
+<div class="container mt-3 col-10">
+<div class="d-flex justify-content-end m-3">
+  <button type="button" class="btn btn-success ml-3" data-toggle="modal" data-target="#addEvent">
+    Add
+    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-plus-square-fill" viewBox="0 0 16 16">
+      <path d="M2 0a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V2a2 2 0 0 0-2-2H2zm6.5 4.5v3h3a.5.5 0 0 1 0 1h-3v3a.5.5 0 0 1-1 0v-3h-3a.5.5 0 0 1 0-1h3v-3a.5.5 0 0 1 1 0z"></path>
+    </svg>
```
Comment (YilKanda) [inline EasyLearn/app/views/events/index.html.erb]: What do you think about passing the SVG code to an external file?

File: EasyLearn/app/views/events/index.html.erb (hunk 2 above)
Comment (javs1) [reply EasyLearn/app/views/events/index.html.erb]: Dome

Comment (javs1) [reply EasyLearn/app/views/events/index.html.erb]: done*

File: EasyLearn/app/views/events/index.html.erb (hunk 4 above)
Comment (javs1) [reply EasyLearn/app/views/events/index.html.erb]: done

File: EasyLearn/app/views/events/index.html.erb (hunk 6 above)
Comment (javs1) [reply EasyLearn/app/views/events/index.html.erb]: done

File: EasyLearn/config/database.yml.example (hunk 5 above)
Comment (javs1) [reply EasyLearn/config/database.yml.example]: done

File: EasyLearn/app/views/events/index.html.erb (hunk 1 above)
Comment (javs1) [reply EasyLearn/app/views/events/index.html.erb]: done

**Pull request merged**